.github/workflows/tests.yml

```diff
@@ -37,26 +37,30 @@ jobs:
       - name: Lint ✨
         run: npm run test:lint
 
-  Build:
+  Release:
     needs: [Test, Lint]
-    if: github.ref == 'refs/heads/master'
+    if: |
+      github.ref == 'refs/heads/master' &&
+      github.event.repository.fork == false
     runs-on: ubuntu-latest
     steps:
       - name: Checkout Code
         uses: actions/checkout@v2
       - name: Install Dependencies
         run: npm ci
       - name: Build 🗜️
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         run: |
           npm run build
           if ! git diff --quiet; then
             git config --global user.email "<>"
             git config --global user.name "MarkedJS bot"
             git commit -am "🗜️ build [skip ci]"
-            git push "https://${GITHUB_ACTOR}:${GITHUB_TOKEN}@github.com/markedjs/marked.git" HEAD:master
           fi
+      - name: Release 🎉
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+        run: npx semantic-release
```

Comment:
```suggestion
        - name: Release 🎉
        uses: cycjimmy/semantic-release-action@v2
        with:
          extra_plugins: |
            @semantic-release/git@9.0.0
            @semantic-release/github@7.1.1
            @semantic-release/npm@7.0.6
            @semantic-release/release-notes-generator@9.0.1
        env:
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
          NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
```

Reply:
Then we would have to manually update the versions when new versions are released instead of relying on tools. I think the small amount of disk space they take up is worth the small amount of developer time it would take with this approach.

Reply:
You don't need to specify the versions. Specifying the names is enough.

Reply:
Then it always installs the latest and can break things if a major version is released. I still think keeping them in package.json with version numbers is better.

Reply:
They are only dev dependencies if you really don't want them installed you can go through the extra work of removing them from package.json before running npm i on your machine.

Reply:
for that matter if you use pnpm they won't take up any extra space if another package uses them.

Reply:
If you are really concerned about ci dependencies you should petition npm to create a `ciDependencies` field for these types of dependencies.

Reply:
I don't think these break anything. The semantic release supports configs as well. The configs wrap these functionalities in a preset. This is similar to `apm-config` that I use in atom-ide-community. I would be surprised if there is not any config for npm.

https://github.com/semantic-release/apm-config#plugins

Reply:
I don't think we want to use unofficial semantic release tooling--it would increase the scope of vulnerabilities and it wouldn't be checked by github's automated vulnerability checks. And it basically gives cycjimmy access to our releases which is not a risk we need to take.